test/api/che.test.ts

```diff
@@ -130,7 +130,7 @@ describe('Che helper', () => {
         .get('/devfile.yaml')
         .replyWithFile(200, __dirname + '/requests/devfile.valid', { 'Content-Type': 'text/plain; charset=utf-8' }))
       .nock(cheURL, api => api
-        .post('/api/devfile')
+        .post('/api/workspace/devfile')
```

Comment:
Maybe use a constant for the API_ENDPOINT

Reply:
:+1: done

Reply:
thx